release/README.md

```diff
@@ -0,0 +1,10 @@
+# Releases
+    
+Releases are created with Docker. Alternatively, directly execute the `./release/release.sh` script. First, create the Docker image:
```

Comment:
I think this explanation is a bit confusing and starts a bit out of nowhere.
I'd propose to start this here with a small explanation on how a release of O-Neko looks like (so naming matching git tags and versions in pom.xml and packae.json).
Then there shoule be a separate section explaining the overall "How do I build the release of say version 1.5.7".
If there are alternatives to some steps then you should not continue in regular text, cause then it is unclear whether this is still part of the alternative way or not.

Reply:
And if your explanation covers the release-script then you totally should mention it's mandatory parameter.